admin-ui/plugins/auth-server/components/AuthN/AuthNForm.js

```diff
@@ -56,9 +55,9 @@ function AuthNForm({ item, handleSubmit }) {
     setModal(!modal)
   }
 
-  const submitForm = () => {
+  const submitForm = (userMessage) => {
     toggle()
-    handleSubmit(formik.values)
+    handleSubmit(formik.values, userMessage)
   }
```

Comment:
Normalize boolean before submit to avoid string leakage

defaultAuthNMethod comes from a <select> and may be "true"/"false" strings. Coerce to boolean before calling handleSubmit.


```diff
-  const submitForm = (userMessage) => {
-    toggle()
-    handleSubmit(formik.values, userMessage)
-  }
+  const submitForm = (userMessage) => {
+    toggle()
+    const values = {
+      ...formik.values,
+      defaultAuthNMethod:
+        formik.values.defaultAuthNMethod === true ||
+        String(formik.values.defaultAuthNMethod).toLowerCase() === 'true',
+    }
+    handleSubmit(values, userMessage)
+  }
```